Kitodo/src/main/webapp/WEB-INF/templates/includes/desktop/projectsWidget.xhtml

```diff
@@ -19,16 +19,20 @@
     <p:dataTable id="projectTable"
                  value="#{DesktopForm.projects}"
                  scrollable="true"
+                 styleClass="default-layout"
                  var="project">
         <p:column headerText="#{msgs.title}"
+                  sortBy="#{project.title}"
```

Comment:
```suggestion
                  sortBy="#{project.title.keyword}"
```

Reply:
The records for the desktop widgets are not retrieved from the index like the normal list views, but instead from the DB, so the index specific `keyword` does not exist here. Instead, sorting projects in the `projectsWidget` should already work without it!